app/services/id_token_builder.rb

```diff
@@ -37,9 +37,9 @@ def jwt_payload
 
   def id_token_claims
     {
-      acr: (acr if !sp_requests_vot?),
-      vot: (vot if sp_requests_vot?),
-      vtm: (IdentityConfig.store.vtm_url if sp_requests_vot?),
+      acr: acr,
+      vot: vot,
+      vtm: IdentityConfig.store.vtm_url,
```

Comment:
I think we can remove these keys entirely because they're "inherited" from how this class merges in `user_info`
```suggestion
```

Reply:
Removed.